### ISSUE TITLE
[Feature Request] There's no proper way to change NavMenu background color on hover

### ISSUE THREAD

**Issue by bittermeatball** [ElemeFE/element#19988]
### Existing Component
Yes

### Component Name
NavMenu

### Description
I think there should be some thing like:
hover-background-color
or something like that, right now, i have to overide the CSS class with !important, and i don't think that's the right way

**Comment by rc-front** [comment]
I agree with you！I have tried to use the native events to change the hovered banckground-color, but something was wrong with it, I found that I can not change the class of el-submenu's background-color
